test/core/security/authorization/test_policies/BUILD

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])
```

Comment:
@ctiller I removed the comment from here based on go/thirdparty/linter#build-licenses-comment
Also verified the change by importing this PR
https://critique-ng.corp.google.com/cl/401032333